components/commands/common/features.cc

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#include "brave/components/commands/common/features.h"
+
+namespace commands::features {
+BASE_FEATURE(kBraveCommandsFeature,
```

Comment:
1. `kBraveCommandsFeature` -> `kBraveCommands`.
2. add a comment in .cc file describing the feature.

please add newlines after/before namespace opening/closing everywhere in this PR. This style is applied everywhere in Chromium (although clang-format doesn't enforce it), but it's easier to read this way:
```
namespace commands::features {

BASE_FEATURE(kBraveCommandsFeature, ...);

}  // namespace commands::features

namespace commands {

std::string GetKeyName(ui::KeyboardCode code);
std::vector<std::string> GetModifierName(ui::KeyEventFlags flags);

}  // namespace commands
```

Reply:
Cool, I much prefer not having `Feature` on the end of our flags but I was cargo culting :smile: